src/blk/BlockDevice.h

```diff
@@ -200,6 +200,11 @@ class BlockDevice {
     return conventional_region_size;
   }
 
+  virtual bool reset_zones(uint64_t zone_num_range_start, uint64_t zone_num_range_end) {
+    ceph_assert(is_smr());
```

Comment:
Not sure if we should assert here given that right below we return false. Let's see what Ceph folks say.